generators/server/templates/src/main/resources/application-dev.yml.ejs

```diff
@@ -103,7 +103,7 @@ logger:
 
 micronaut:
   server:
-    port: <%= serverPort %>
```

Comment:
Mostly not relevant for the ci pipeline, but we should keep <%= serverPort %> it is supposed to come from the main generator